+++
# Date this page was created.
date = "2016-04-27"

# Project title.
title = "Chromosomal Inversions and Adaptation"

# Project summary to display on homepage.
summary = "What role do chromosomal inversions play in biological adaptation?"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "inversion_image.png"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["adaptation", "recombination", "inversions"]

# Does the project detail page use math formatting?
math = false

+++

Chromosomal inversions often underlie local adaptation and reproductive isolation and are widely believed to evolve via natural selection.Yet, in spite of a long history of study, the specific role chromosomal inversions play during adaptation remains unclear. 

A key model authored by Kirkpatrick & Barton posits that inversions can be favored by selection if they create tightly-linked “cassettes” of locally-adapted alleles that act as single alleles of large effect and are shielded from maladaptive gene flow. 

The core prediction of this model is that maladaptive gene flow generates a fitness differential between inverted and non-inverted haplotypes, which results in the inverted haplotype evolving to high frequency. 

As part of my NSF-funded research at Duke University, I am working on an experimental test of this idea. Our two aims are:

1. Create several novel chromosomal inversions using modern transgenic approaches.
2. Perform an experimental test of the role of maladaptive gene flow in driving the adaptive evolution of these inversions.
